spell.txt

```diff
@@ -0,0 +1,4 @@
+Appwrite
```

Comment:
What does this file use for?

Reply:
So sorry, wasn't mean to commit that 😢  Removed it